source/sql/requirements.txt

```diff
@@ -4,5 +4,5 @@ Mako==0.9.1
 MarkupSafe==0.18
 SQLAlchemy==1.3.3
 Shapely==1.3.0
-alembic==0.6.2
-psycopg2
```

Comment:
Shouldn't we fix the version of psycopg2 ?

Reply:
we want to use the psycopg installed by the system to not have rebuild everything